schema.json

```diff
@@ -0,0 +1,496 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "required": [
+        "format_version",
+        "generated_at_timestamp",
+        "steplib_source",
+        "download_locations",
+        "assets_download_base_uri",
+        "steps"
+    ],
+    "additionalProperties": false,
+    "properties": {
+        "format_version": {
+            "type": "string",
+            "pattern": "^\\d+.\\d+.\\d+$",
```

Comment:
please test this for `0.0.0` and:
(multi line)
```
1.0.0
1.0.0
```

Reply:
dot is not escaped

Reply:
multi-line handled since the first and last defined (`^..$`)